README.md

```diff
@@ -43,7 +43,58 @@ Differences with rust-mode:
 
 # Installation
 
-Simply put `(use-package rustic)` in your config and most stuff gets configured automatically. 
+## From MELPA
+
+* Make sure you have [added MELPA repos](https://melpa.org/#/getting-started).
+* `M-x package-install RET rustic RET`.
+
+## From source
+
+* Clone this git repo somewhere (e.g. `~/prog/rustic`).
+* Add the path to Emacs `load-path` and associate .rs files with
+  rustic by adding this to your init file:
+
+```
+(add-to-list 'load-path "~/prog/rustic")
+(autoload 'rustic-mode "rustic")
+(add-to-list 'auto-mode-alist '("\\.rs\\'" . rustic-mode))
+```
+
+# Completion
+
+Auto-completion is optionnal and can work with multiple frontends and
+backends (see rest of document). To keep this simple, we will use the
+default basic setup here. You will need to:
+
+* Install additional Emacs packages: `lsp-mode` and `company`.
+* Install additionnal Rust components with `rustup component add rls rustfmt rust-src`.
+* Load and configure all of it:
+
+```
+(defun my-rust-hook ()
+  "This is run on every rust file open"
+  ;; enable company if it wasnt already
+  (require 'company)
+  (company-mode 1)
+
+  ;; bind "force completion request" to key
+  (local-set-key (kbd "<C-return>") 'company-complete))
+
+(eval-after-load 'rustic
+  '(progn
+     (require 'lsp) ;; error without this
```

Comment:
This shouldn't happen. rustic even offers to install lsp-mode if it isn't installed already. I have to check why this doesn't work anymore.

Reply:
Should be fixed with the latest commit. Please try if it works now.